docs/user/transitioning_from_defaults.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Transitioning from defaults
+---
+
+# Transitioning from Anaconda's `defaults` channels
+
+Users wishing to transition from Anaconda's `defaults` (i.e. those hosted under
+https://repo.anaconda.com/pkgs) channels should do so with care so as not to
+break their working environments. The first thing to realize
+is that in 2024 conda-forge is incompatible with the packages provided in
+`defaults`. Often, the packages on conda-forge will require
+newer versions of a particular package, or, may simply be organized in a
+different fashion. The only workflow we can support as a community is one where
+the conda-forge channel takes priority over all other channels.
+
+If you were using Miniconda or Anaconda Distribution and wish to move to use
+conda-forge, we outline two workflows. The first allows you to try packages from
+conda-forge without uninstalling your previous installation, the second
+involves uninstalling Miniconda or Anaconda Distribution, and then installing
+Miniforge.
+
+## Trying conda-forge in an isolated environment
+
+1. Create a conda environment with very few dependencies. The following
+   command will only use packages from `conda-forge` to install Python 3.11. You
+   may adjust the version of Python to the one of your liking:
+
+   ```bash
+   conda create --name conda-forge-env python=3.11 --channel conda-forge --override-channels
```

Comment:
Super minor comment but this would make it less "dated" and folks reading won't think these instructions are too old.

```suggestion
   conda create --name conda-forge-env python=3 --channel conda-forge --override-channels
```